scripts/test-add-model.ts

```diff
@@ -0,0 +1,35 @@
+import {
+  awaitJob,
+  enqueueAddModel,
+  getApiUrl,
+  getAuthHeaders,
+  runIfCalledAsScript
+} from 'scripts/shared';
+
+async function main(model?: string) {
+  const modelName = model || process.env.MODEL!;
+  console.log(`Test adding model ${modelName}`);
+  const body = await enqueueAddModel(modelName);
+  console.log('Successfully queued model to add', body);
+
+  // fetch job success or failure for up to the timeout of 1 hour
+  const timeoutMs = 60 * 60 * 1000;
+  await awaitJob(body.job_id, timeoutMs);
```

Comment:
nit: Can we think of an alternative name that doesn't repeat the `await`?

Reply:
went with `pollForJobCompletion`